go.mod

```diff
@@ -12,10 +12,9 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/sns v1.7.0
 	github.com/aws/aws-sdk-go-v2/service/sts v1.6.0
 	github.com/aws/smithy-go v1.8.1
+	github.com/clumio-code/clumio-go-sdk v0.0.1-rc1
```

Comment:
Does `rc1` here mean the SDK is still a release candidate? If so, then if the provdier is referencing the SDK, should the SDK go GA?

Reply:
Yes. It will change once we make the go sdk repository public. As of now it is internal. Have to work with Stephane to make it public. When me go public we will release version 0.1.0 and that is what we will reference in the provider. Also without the SDK being released we can't really merge the current changes as compilation will fail because the sdk will not be available due to it being in a private repository. So that is the first priority next week.

Reply:
Got it thanks, if we don't have a JIRA already for making the SDK public can we make one?